lucetc/Cargo.toml

```diff
@@ -9,22 +9,19 @@ categories = ["wasm"]
 authors = ["Lucet team <lucet@fastly.com>"]
 edition = "2018"
 
-[lib]
-crate-type=["rlib"]
```

Comment:
Doesn't actually change anything; this is the implicit default behavior for a manifest with a `src/lib.rs`